docs/security-considerations.rst

```diff
@@ -273,17 +290,17 @@ control to a fixed and trusted third party or just converts the contract into
 a simple "give me back my money" contract.
 
 
-*******************
-Formal Verification
-*******************
+Method visibility from mother contracts and ABI
+===============================================
+
+
+When compiling a contract that inherits from other contracts, the generated JSON ABI for the child contract does not contain all the methods from contracts in the upper hierarchy. However, that does not prevent public methods from being executed. You should take care of which methods can be executed. Use `internal` function modifier to control visibility.
```

Comment:
Yes, I've created this in order to make people aware of the issue that caused Parity's multisig bug. Even though the method `initWallet` was available, it took so long to people realize because it wasn't in the ABI (checked on etherscan). But yes, that should be about the Library.

Any thoughts on how this could be improved?